src/code42cli/output_formats.py

```diff
@@ -76,6 +77,30 @@ def _requires_list_output(self):
         return self.output_format in (OutputFormat.TABLE, OutputFormat.CSV)
 
 
+class DataFrameOutputFormatter:
+    def __init__(self, output_format):
+        output_format = output_format.upper() if output_format else OutputFormat.TABLE
+        self.output_format = output_format
+        self._format_func = DataFrame.to_string
+        self._output_args = {}
```

Comment:
We should add `index=False` to all these output args so we're not printing/adding the dataframe index values.